doc/Financial.md

```diff
@@ -0,0 +1,105 @@
+# How to run the SPA with the Financial-grade Token Handler
+
+## Overview
+
+The token handler uses the following behaviour:
+
+- OAuth work is done for the SPA by an API
+- Authorization Code Flow (PKCE) is used, along with a Mutual TLS client secret
+- [PAR](https://tools.ietf.org/id/draft-lodderstedt-oauth-par-00.html) and [JARM](https://openid.net/specs/openid-financial-api-jarm.html) are also used, as state-of-the-art security features
+- Only the strongest `SameSite=strict` cookies are used in the browser
+- The code example uses HTTPS for all components
+
+## Configure Development Domains
+
+Add these entries to your /etc/hosts file:
+
+```bash
+127.0.0.1 localhost www.example.com api.example.com login.example.com
+:1        localhost
+```
+
+## Install Prerequisites
+
+Ensure that these tools are installed locally:
+
+- [Node.js](https://nodejs.org/en/download/)
+- [Java 11 or later](https://openjdk.java.net/projects/jdk/11/)
```

Comment:
Next step could be running all components in containers which have Java and node, when necessary. This would then require the user to only have docker desktop and jq.